.github/workflows/test-harness-javascript-ariesvcx.yml

```diff
@@ -37,11 +37,6 @@ jobs:
           TEST_SCOPE: "-t @RFC0036,@RFC0037,@RFC0160 -t ~@T003-RFC0160 -t ~@T004-RFC0160 -t ~@revocation -t ~@RFC0025 -t ~@RFC0183 -t ~@RFC0211 -t ~@RFC0434 -t ~@RFC0453 -t ~@wip -t ~@DIDExchangeConnection -t ~@QualifiedDIDs"
           REPORT_PROJECT: javascript-b-aries-vcx
         continue-on-error: true
-      - uses: actions/upload-artifact@v4
-        with:
-          name: agent-logs
-          path: ./test-harness/.logs/
-      - name: run-send-gen-test-results-secure
```

Comment:
@swcurran, It looks like you deleted one too many lines here.

Reply:
Doh…good catch. Fixed.